core/storage/statistics/gae_models.py

```diff
@@ -728,6 +717,61 @@ def create(
         return entity_id
 
 
+class StateFinishEventLogEntryModel(base_models.BaseModel):
+    """An event triggered by a student finishing a state.
+
+    Event schema documentation
+    --------------------------
+    V1:
+        event_type: 'state_finish'
+        exp_id: id of exploration currently being played
+        exp_version: version of exploration
+        state_name: Name of current state
+        client_time_spent_in_secs: Time since start of this state when this
+            event was recorded
+        session_id: ID of current student's session
+    """
+    # Which specific type of event this is
+    event_type = ndb.StringProperty()
```

Comment:
Actually, I just realized, why do we even need this? It's the same for all model instances in this class, right? Is it ever used?

Reply:
@seanlip Yep, no point having them in the storage layer.

Reply:
Done.